### PR TITLE
add driver and germline OQL documentation

### PR DESCRIPTION
**Pull request by tmazor**

### CODE REVIEW

File: docs/Onco-Query-Language.md [hunk 1]
```diff
@@ -134,6 +162,71 @@ Or over-expressed at the phospho-protein level:
...
+* `DRIVER`
+
+To see all germline BRCA1 mutations:
+```
+BRCA1: GERMLINE
+```
+
+Or to see specifically truncating germline mutations:
+```
+BRCA1: TRUNC_GERMLINE
```
Comment (adamabeshouse) [inline docs/Onco-Query-Language.md]: (or GERMLINE_TRUNC) ?

File: docs/Onco-Query-Language.md [hunk 2]
```diff
@@ -134,6 +162,71 @@ Or over-expressed at the phospho-protein level:
...
+```
+
+Or to see somatic missense mutations:
+```
+BRCA1: MUT = MISSENSE_SOMATIC
+``` 
+
+When combining `GERMLINE` or `SOMATIC` with `DRIVER` and another term (e.g. `NONSENSE`), it is essential that the middle term is `GERMLINE` or `SOMATIC` or `DRIVER`; `GERMLINE_NONSENSE_DRIVER` will not work.
```
Comment (adamabeshouse) [inline docs/Onco-Query-Language.md]: I find it confusing to explain this in terms of the middle term... would it be clearer to say "the mutation term must appear at the beginning or the end"?

Comment (jjgao) [reply docs/Onco-Query-Language.md]: @adamabeshouse is it possible to support any order?

Comment (adamabeshouse) [reply docs/Onco-Query-Language.md]: yes, ill look into that

Comment (adamabeshouse) [reply docs/Onco-Query-Language.md]: @tmazor we're now supporting any order so this line isn't necessary.

Comment (tmazor) [reply docs/Onco-Query-Language.md]: Great - I updated the page accordingly @adamabeshouse

**Pull request merged**